tests/Main.hs

```diff
@@ -82,17 +87,47 @@ main = hspec $ do
             rp <-  convertGeometryToRaw $ PolygonGeometry polygon1 Nothing
             R.getTypeId rp
       t `shouldBe` 3
+    it "Converts a MultiPolygon from Raw" $ do
+      let (x,y) = runGeos $ do
+            r <- RS.createReader
+            mpr <- RS.readHex r multiPolygonStringBS
+            MultiPolygon vps <- convertMultiPolygonFromRaw mpr
+            let (Polygon vlr) = vps V.! 0
+                (LinearRing vc) = vlr V.! 0
+                (Coordinate2 x y) = vc V.! 0
+            return (x, y)
+
+      x `shouldBe` 153.160525
+      y `shouldBe` -27.377412
     it "Tests disjoint" $ do
-      (disjoint (PolygonGeometry polygon1 Nothing) (PolygonGeometry polygon2 Nothing)) `shouldBe` False 
+       (disjoint (PolygonGeometry polygon1 Nothing) (PolygonGeometry polygon2 Nothing)) `shouldBe` False 
     it "Tests intersects" $ do
-      intersects (PolygonGeometry polygon1 Nothing) (PolygonGeometry polygon2 Nothing) `shouldBe` True
+       intersects (PolygonGeometry polygon1 Nothing) (PolygonGeometry polygon2 Nothing) `shouldBe` True
+    it "Tests raw parsing" $ do
+        let (x,y) = runGeos $ do
+                r <- RS.createReader
+                g <- RS.readHex r multiPolygonStringBS
+                gi :: R.GeomConst <- R.getGeometryN g 0
+                ir :: R.GeomConst <- R.getExteriorRing gi
+                cs :: RC.CoordSeqConst <- R.getCoordinateSequence_ ir
+                x <- RC.getCoordinateSequenceX cs 0
+                y <- RC.getCoordinateSequenceY cs 0
+                return (x,y)
+        x `shouldBe` 153.160525
+        y `shouldBe` -27.377412
 
   describe "Tests Serialization" $ do
     it "Parses a bytestring to a linestring" $  do
       let lsg :: Geometry LineString = withSomeGeometry (readHex linestringBS) $ \case
                   ls@(LineStringGeometry _ _) -> ls
                   _ -> error "asda"
       lsg `shouldBe` linestring
+    it "Parse a multipolygon" $ do
+      pending
+      let multig :: Geometry MultiPolygon = withSomeGeometry (readHex multiPolygonStringBS) $ \case
+                  g@(MultiPolygonGeometry _ _) -> g
+                  _ -> error "asda"
+      (show multig) `shouldBe` (show linestring)
```

Comment:
@jamiecook Multipolygon and Linestring have different show instances. We need to compare the results with an actual MultiPolygon object.